website/dbt-versions.js

```diff
@@ -0,0 +1,19 @@
+exports.versions = [
+    "1.2",
+    "1.0",
+    "0.21",
+    "0.20"
+]
+
+exports.versionedPages = [
```

Comment:
In case others are confused at first, as I was:
- These are for pages documenting functionality are _entirely_ unsupported in older (or newer) versions of dbt
- It's not necessary to add every page to this list, every time there's a small difference in functionality across versions. That's what the `<VersionBlock>` is for

Reply:
That is spot on, thanks for noting that here for others looking over it. I have an outline of documentation for the versioning [in progress here](https://www.notion.so/dbtlabs/Versioning-on-Docusaurus-c6a4a41a66cd4ea2970854cc42cb5b70), and these notes will definitely be great to have in there as well.